fluent/src/context.js

```diff
@@ -210,14 +212,6 @@ export class MessageContext {
   }
 
   _memoizeIntlObject(ctor, opts) {
-    const cache = this._intls.get(ctor) || {};
-    const id = JSON.stringify(opts);
-
-    if (!cache[id]) {
-      cache[id] = new ctor(this.locales, opts);
-      this._intls.set(ctor, cache);
-    }
-
-    return cache[id];
+    return this._intlMemoizer.get(ctor, this.locales, opts);
```

Comment:
Let's remove this method completely and use `ctx._intlMemoizer` directly in the resolver (in `types.js`).